### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request make use of webpack+npm instead of webassets+bower.

Theme handling is a little bit refactore (simplified).

This is currently a work in progress

### CODE REVIEW

File: js/dataset/display.js [hunk 1]
```diff
@@ -174,6 +174,7 @@ define([
                     layer.openPopup();
                 });
                 layer.on("mouseout", function () {
+                    console.log('l', layer, layer.closePopup)
```
Comment (davidbgk) [inline js/dataset/display.js]: Still useful?

Comment (noirbizarre) [inline js/dataset/display.js]: Not at all :)
Was just trying to debug and forgot to remove the line

File: js/apidoc.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    });
+
+    var swaggerUi = new SwaggerUi({
+        url: $('meta[name="swagger-specs"]').attr('content'),
+        dom_id: "swagger-ui-container",
+        supportedSubmitMethods: ['get'],
+        onComplete: function(swaggerApi, swaggerUi){
+            log.debug("Loaded SwaggerUI");
+
+            // if(typeof initOAuth == "function") {
```
Comment (davidbgk) [inline js/apidoc.js]: Useful?

File: js/dataset/display.js (hunk 1 above)
Comment (davidbgk) [inline js/dataset/display.js]: Bump :)

File: js/home.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+// require('raven');
+
+var $ = require('jquery');
+require('bootstrap');
+
+// require('site');
+
+$('.carousel').carousel();
+
+// var Vue = require('vue'),
```
Comment (davidbgk) [inline js/home.js]: Remove below

File: js/site.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+// require('raven');
+
+var $ = require('jquery');
+require('bootstrap');
+
+require('common');
+
+
+
+// var Vue = require('vue'),
```
Comment (davidbgk) [inline js/site.js]: Remove below, `déjà vu` impression with `home.js`?

Comment (noirbizarre) [inline js/site.js]: I will remove it. Copy-paste evil as you said ;)

**Comment by davidbgk** [comment]
r+

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    module: {
+        loaders: [
+            {test: /\.(jpg|jpeg|png|gif|svg)$/, loader: 'file'},
+            {test: /\.css$/, loader: css_loader},
+            {test: /\.less$/, loader: less_loader},
+            {test: /\.vue$/, loader: vue.withLoaders({css: css_loader, less: less_loader})},
+            {test: /\.json$/, loader: "json"},
+            {test: /\.hbs$/, loader: 'handlebars?debug=true&helperDirs[]=' + path.join(__dirname, 'js', 'templates', 'helpers')},
+            {test: /\.html$/, loader: 'html'},
+            {test: /\.(woff|svg|ttf|eot|otf)([\?]?.*)$/, loader: "file-loader?name=[name].[ext]"},
```
Comment (davidbgk) [inline webpack.config.js]: `woff2` too?

Comment (noirbizarre) [inline webpack.config.js]: It's already taken in account by the regex

File: js/site.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+require('../less/site.less');
+
+// Catch all errors
+// require('raven');
+
+var $ = require('jquery');
+require('bootstrap');
+
+require('common');
+require('expose?uData!udata');
```
Comment (davidbgk) [inline js/site.js]: :+1:

**Pull request merged**